src/FileDownloader.php

```diff
@@ -7,6 +7,7 @@
 use GuzzleHttp\Client;
 use Phpcq\Exception\InvalidHashException;
 use Phpcq\Exception\RuntimeException;
+use function file_get_contents;
```

Comment:
This is either superfluous or `file_put_contents` is missing.

Reply:
Both is used but only one imported. Time to get our coding standard ready so that `phpcs` will fix it. @baumannsven ;-)

Reply:
Changed in 7de3179.